app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt

```diff
@@ -78,7 +78,7 @@ class AdministratorControlsFragmentPresenter @Inject constructor(
   private fun createRecyclerViewAdapter(isMultipane: Boolean):
     BindableAdapter<AdministratorControlsItemViewModel> {
       return BindableAdapter.MultiTypeBuilder
-        .newBuilder<AdministratorControlsItemViewModel, ViewType> { viewModel ->
+        .Factory(fragment).create<AdministratorControlsItemViewModel, ViewType> { viewModel ->
```

Comment:
This factory should be injected instead of constructed here.

Reply:
Yeah noted have noticed i still missed some classes during refactor. Will have to run global search to cover all required classes.

Reply:
FWIW changing the class visibilities as suggested below should trigger build failures for any case where this hasn't been corrected, so an audit may not be necessary.